src/main/java/org/molgenis/vcf/report/mapper/HtsJdkToPersonsMapper.java

```diff
@@ -52,11 +54,12 @@
   }
 
   private Person createPerson(String sampleName, Integer offset) {
-    return Person.newBuilder()
-        .setIndividualId(sampleName)
-        .setPaternalId(MISSING_PERSON_ID)
-        .setMaternalId(MISSING_PERSON_ID)
-        .setFamilyId(MISSING + offset)
-        .build();
+    return new Person(
```

Comment:
offtopic: we should consider using Lombok builders to prevent bugs such as switching paternal and maternal identifiers.